### ISSUE TITLE
Server crash when slender_containment breach gets picked

### ISSUE THREAD

**Issue by BimboBumboBambo** [DFS-Servers/BugReportsAndSuggestions#176]
**Steam Profile**
https://steamcommunity.com/id/BimboBumboBambo.

**Describe the bug**
As the title suggests, the server seems to crash whenever the next map is slender_cb. It's unknown if any of the other slender maps does this, but it's unlikely.
**EDIT: It seems to be a targetting issue instead of a map issue.**

**To Reproduce**
Get targeted.

**Expected behavior**
I hope this bug gets fixed instantly. The whole server suffers from the crashes and can extend the load time.

**Screenshots or error logs**
![crash](https://user-images.githubusercontent.com/41446589/62877662-d4054480-bd27-11e9-97bb-213acd1ed642.png)
**EDIT: Proof of crash is still there, but it has nothing to do with the map.**

**Server Info (please complete the following information):**
 - Region: EU
 - IP 89.35.29.11:27145

**Additional context**
The server does go back to tm_stadium after the crash.

*Thanks to Pro100RedMushroom for testing it out more proper.*

**Comment by Pro100RedMushroom** [comment]
after some tetsting on us, i can conclude that the issue is not with the map, but with the bosses.
when we went to slender frost run, everything was fine until a boss spotted a player. after that it crashed.

tl;dr it seems to be a targetting issue

**Comment by Dispenz0r** [comment]
The crashes related to SF2 should now be completely fixed.

I've given you a Community Box now, thanks for letting us know!
